python_modules/dagster/dagster/core/object_store.py

```diff
@@ -1,6 +1,14 @@
 import os
 import pickle
 
+from io import BytesIO
+
+try:
```

Comment:
what would you think about doing this include on-demand within S3ObjectStore methods and then failing in a loud way?

Reply:
happy to do that